rules/opentitan_test.bzl

```diff
@@ -310,17 +310,20 @@ def opentitan_functest(
         elif target in ["cw310_rom", "cw310_test_rom"]:
             devices_to_build_for.append("fpga_cw310")
 
+            # If the user passed in a bitstream, record it here so that we do
+            # not override it later
+            user_bitstream = cw310.get("bitstream", None) if cw310 else None
```

Comment:
Nit: could just say `cw310.get("bitstream")` since the default value for the `default` parameter is `None`.